orttraining/orttraining/training_api/include/utils.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+#include "core/graph/model.h"
+#include "core/session/inference_session.h"
+#include "core/providers/cpu/cpu_execution_provider.h"
+namespace onnxruntime {
+namespace training {
+namespace api {
+namespace utils {
+
+// TODO: consolidate the gradient names with frontend tooling
+const std::vector<std::string> GRAD_SUFFIX{"_grad.accumulation.buffer", "_grad", "_grad.accumulation.out"};
```

Comment:
if they only are used by optimizer, I would suggest putting here in optimizer.h/cc.